File: app/src/main/java/com/yeputra/moviecatalogue/utils/InfoUtils.java
package com.yeputra.moviecatalogue.utils;

import android.app.Activity;
import android.content.Context;
import android.widget.Toast;

import com.google.android.material.snackbar.Snackbar;

public class InfoUtils {
    static void toast(Context ctx, String message) {
        Toast.makeText(ctx, message, Toast.LENGTH_SHORT).show();
    }

    static void snackbar(Activity ctx, String message) {
        Snackbar.make(ctx.getWindow().getDecorView(), message, Snackbar.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/yeputra/moviecatalogue/utils/InfoUtils.kt
package com.yeputra.moviecatalogue.utils

import android.app.Activity
import android.content.Context
import android.widget.Toast
import com.google.android.material.snackbar.Snackbar

fun Context.toast(message: String) {
    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
}

fun Activity.snackbar(message: String) {
    Snackbar.make(window.decorView, message, Snackbar.LENGTH_SHORT).show()
}
